Source/Models/FORMSection.m

```diff
@@ -59,6 +59,7 @@ - (instancetype)initWithDictionary:(NSDictionary *)dictionary
         field.sectionSeparator = YES;
         field.position = @(fields.count);
         field.section = self;
+        field.fieldID = [NSString stringWithFormat:@"separator-%@", _sectionID];
```

Comment:
Yah, I think I saw the `description` method exploding when running it on a separator.